src/containers/Layout/Footer.tsx

```diff
@@ -0,0 +1,48 @@
+import {
+  PageHeader,
+  PageHeaderTools,
+  PageHeaderToolsItem
+} from "@patternfly/react-core";
+import React from "react";
+import { useHistory } from 'react-router-dom';
+import logo from "../../assets/images/logo-white.png";
+import github from "../../assets/images/githubLogo.png";
```

Comment:
Just something to consider: I noticed that the shade of blue used for the GitHub logo isn't the same as the primary blue colour used by PatternFly (`#06c`). Although, I'm not sure how strict we need to be with our colour scheme since it seems our `<a>` elements all use different shades of blue.

Reply:
Yeah, i agree. the colour for the github blue isn't a perfect match, but it's nearly impossible to find an appropriate logo/icon that matches the colour scheme perfectly. I think that it looks extremely close enough to have the UI consistent.

And yeah, I also noticed that the shades of blue for the anchors are a little different as well, I'd say this is an issue to address later when refactoring the UI